code/addons/vitest/src/components/TestProviderRender.tsx

```
@@ -319,63 +317,66 @@ export const TestProviderRender: FC<TestProviderRenderProps> = ({
+                padding="small"
+                size="medium"
+                variant="ghost"
+                ariaLabel={
+                  // FIXME: I can't deduce from the original tooltip logic whether this use case
+                  // is logically possible or not. It is a reachable conditional branch in the original code.
+                  isRunning
+                    ? 'Open coverage report (testing still in progress)'
+                    : `Open coverage report (${currentRun.coverageSummary.percentage}% coverage)`
+                }
+              >
+                <a href="/coverage/index.html" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)